### PR TITLE
Added the function of sending SMS messages through Alibaba Cloud.

### PR DESCRIPTION
**Pull request by lwqzz**
## Checklist

- [ ]  I have read the [Contributing Guide](https://hertzbeat.apache.org/docs/community/code_style_and_quality_guide)
- [ ]  I have written the necessary doc or comment.
- [ ]  I have added the necessary unit tests and all cases have passed.

## Add or update API

- [ ] I have added the necessary [e2e tests](https://github.com/apache/hertzbeat/tree/master/e2e) and all cases have passed.

### CODE REVIEW

File: common/src/main/java/org/apache/hertzbeat/common/config/CommonProperties.java [hunk 1]
```diff
@@ -227,6 +238,73 @@ public String getTemplateId() {
...
 
+        public void setTemplateId(String templateId) {
+            this.templateId = templateId;
+        }
+    }
+    public static class AliYunSmsProperties {
+
+        /**
+         * 阿里云账户访问密钥id
+         */
```
Comment (tomsun28) [inline common/src/main/java/org/apache/hertzbeat/common/config/CommonProperties.java:249]: 👍 hi, please use the english describe.

**Review by tomsun28 — COMMENTED** [review]
👍👍

File: common/src/main/java/org/apache/hertzbeat/common/config/CommonProperties.java (hunk 1 above)
Comment (lwqzz) [reply common/src/main/java/org/apache/hertzbeat/common/config/CommonProperties.java:249]: Should all the descriptions inside be written in English?

**Comment by tomsun28** [comment]
please add the apache header in files

File: common/src/main/java/org/apache/hertzbeat/common/service/AliYunSmsClient.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * sms service client for aliyun cloud
+ * @author lwq
```
Comment (tomsun28) [inline common/src/main/java/org/apache/hertzbeat/common/service/AliYunSmsClient.java:38]:
```suggestion
```
hi, we will not add author in file header.

File: common/src/main/java/org/apache/hertzbeat/common/util/AliYunSendSMS.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+ */
+package org.apache.hertzbeat.common.util;
+
+import com.aliyun.dysmsapi20170525.models.SendSmsRequest;
+import com.aliyun.dysmsapi20170525.models.SendSmsResponse;
+import com.aliyun.teaopenapi.models.Config;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.Map;
+
```
Comment (tomsun28) [inline common/src/main/java/org/apache/hertzbeat/common/util/AliYunSendSMS.java:25]: please add head describe here

File: manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/impl/AliYunAlertNotifyHandlerImpl.java [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+import org.apache.hertzbeat.common.service.AliYunSmsClient;
+import org.apache.hertzbeat.common.util.ResourceBundleUtil;
+import org.apache.hertzbeat.manager.support.exception.AlertNoticeException;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Component;
+
+import java.util.ResourceBundle;
+
+/**
+ * @author lwq
```
Comment (tomsun28) [inline manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/impl/AliYunAlertNotifyHandlerImpl.java:35]:
```suggestion
 * alicloud alarm handler implment
```

File: common/src/main/java/org/apache/hertzbeat/common/service/AliYunSmsClient.java [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+        SendSmsRequest req = new SendSmsRequest();
+        req.setSmsSdkAppId(appId);
+        req.setSignName(signName);
+        req.setTemplateId(templateId);
+        req.setTemplateParamSet(templateValues);
+        req.setPhoneNumberSet(phones);
+        try {
+            Map<String, Object> param = new HashMap<>();
+            //taskName：监控名称alert：告警级别 message：告警信息 sysTime：系统时间
```
Comment (tomsun28) [inline common/src/main/java/org/apache/hertzbeat/common/service/AliYunSmsClient.java:94]:
```suggestion
            // taskName: monitoring name,  alert: alarm level, message: alarm content, sysTime:system time
```

File: manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/impl/AliYunAlertNotifyHandlerImpl.java [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+@ConditionalOnProperty("common.sms.aliyun.app-id")
+final class AliYunAlertNotifyHandlerImpl extends AbstractAlertNotifyHandlerImpl {
+
+    private final AliYunSmsClient aliYunSmsClient;
+
+    private final ResourceBundle bundle = ResourceBundleUtil.getBundle("alerter");
+
+    @Override
+    public void send(NoticeReceiver receiver, NoticeTemplate noticeTemplate, Alert alert) {
+        // SMS notification 短信通知
```
Comment (tomsun28) [inline manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/impl/AliYunAlertNotifyHandlerImpl.java:49]:
```suggestion
        // SMS notification 
```

**Review by tomsun28 — COMMENTED** [review]
Hi, the first PR may be a bit troublesome, please fix it.

**Comment by zqr10159** [comment]
image

Hi, please fix them above, we have style check for codes.

**Review by zqr10159 — APPROVED** [review]
lgtm

**Pull request merged**